witness/README.md

```diff
@@ -46,3 +75,8 @@ defined above.  Currently it contains implementations in the following languages
   This can be deployed as a smart contract on the Ethereum blockchain.  There are 
   two main contracts, each of which supports one of the consistency checks 
   described above.
+
+## Feeders
+
+* [Go SumDB](https://github.com/google/trillian-examples/tree/master/sumdbaudit/witness)
+* [Serverless Logs](https://github.com/google/trillian-examples/tree/master/serverless/cmd/feeder)
```

Comment:
nit: `Logs` -> `logs` for case consistency with the rest of this page.

Reply:
I removed the word because I added explainer text to address another comment.